packages/theme-addresses/theme-addresses.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * CountryProvinceSelector Constructor
+ * @param {Object} countryOptions Country JSON object
+ */
+export function CountryProvinceSelector(countryOptions) {
+  this.countryOptions = countryOptions;
+}
+
+/**
+ * Helper function that builds the province selector
+ */
+function buildProvince (provinceNodeElement, provinces) {
+  var defaultValue = provinceNodeElement.getAttribute('data-default');
+
+  provinces.forEach(function (option) {
+    var optionElement = document.createElement('option');
+    optionElement.value = option[0];
+    optionElement.textContent = option[1];
+
+    provinceNodeElement.appendChild(optionElement);
+  })
+  provinceNodeElement.value = defaultValue;
+}
+
+/**
+ * Helper function that determines if province selector needs to be build
+ */
+function resolveProvinces (countryNodeElement, provinceNodeElement, selectedValue, options) {
+  var selectedOption = countryNodeElement.querySelector('option[value="' + selectedValue +'"]');
+  var provinces = JSON.parse(selectedOption.getAttribute('data-provinces'));
+
+  provinceNodeElement.innerHTML = '';
+
+  if (provinces.length) {
+    buildProvince(provinceNodeElement, provinces)
+    options.hideClass && provinceNodeElement.classList.remove(options.hideClass);
+  } else {
+    options.hideClass && provinceNodeElement.classList.add(options.hideClass);
+  }
+}
+
+/**
+ * Builds the country and province selector with the given node element
+ * @param {Node} countryNodeElement The select Node element for country
+ * @param {Node} provinceNodeElement The select Node element for province
+ * @param {Object} options Additional settings available
+ *   hideClass        - The classname that will be toggled for province selector
```

Comment:
`@param {string} option.hideClass - The classname that will be toggled for province selector`